### DISCUSSION TITLE
默认下载地址/coding下载地址均返回错误

### DISCUSSION THREAD

**Discussion by stalkercn** [liuweiGL/vite-plugin-mkcert#53] [General]
默认设置下(应该是github source)会出现Error: Request failed with status code 503
把source修改为coding之后会出现Error: Request failed with status code 400  国内联通网络

**Comment by liuweiGL** [comment]
好的，晚点看看

**Comment by stalkercn** [comment]
可能是我运行环境的问题   在electron/node环境下使用axios有一些问题   浏览器环境应该没事

**Comment by liuweiGL** [comment]
嗯，我这边测试没问题
